actionpack/test/dispatch/cookies_test.rb

```diff
@@ -278,6 +278,21 @@ def noop
     def encrypted_cookie
       cookies.encrypted["foo"]
     end
+
+    def set_expiration_using_expires_in
+      cookies["user_name"] = { value: "assain", expires_in: 1.hour }
+      head :ok
+    end
+
+    def set_expiration_using_expires_at
+      cookies["user_name"] = { value: "assain", expires_at: Time.now.advance(years: 1)}
+      head :ok
+    end
+
+    def set_signed_cookie_expiration_using_expires_in
+      cookies.signed["user_name"] = { value: "assain", expires_in: 1.hour }
+      head :ok
```

Comment:
When switching to `expires` supporting both durations and time we only need to test signed cookies and how their expiry is enforced.

E.g. do as this test but add `travel_to <after_expiry>` and `assert_nil` on the signed cookie.